docs/designers-developers/developers/tutorials/block-tutorial/creating-dynamic-blocks.md

```diff
@@ -189,14 +189,14 @@ registerBlockType( 'gutenberg-examples/example-05-dynamic', {
 	icon: 'megaphone',
 	category: 'widgets',
 
-	edit: function( props ) {
+	edit: ( props ) => {
 		return (
 			<ServerSideRender
 				block="gutenberg-examples/example-05-dynamic"
 				attributes={ props.attributes }
 			/>
 		);
-	},
+	}
```

Comment:
This should stay as is, it follows the WordPress coding standards for ESNext codebase.